src/i18n/strings/en_EN.json

```diff
@@ -444,6 +444,7 @@
         "camera": "Camera",
         "cameras": "Cameras",
         "capabilities": "Capabilities",
+        "conferences": "Conferences",
```

Comment:
I'm not sure this string should be in `common` as it likely has different translations in different contexts (video conference vs an in-person conference for example)